concordium-sdk/src/main/java/com/concordium/sdk/responses/bakerlist/BakerId.java

```diff
@@ -0,0 +1,39 @@
+package com.concordium.sdk.responses.bakerlist;
+
+import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Short id of the baker.
```

Comment:
I'm not sure what 'short' means here.
It is simply the account index of the account baking i.e. the account registered as a baker.

Reply:
changed. done